smarty/tests/test_sampler.py

```diff
@@ -57,14 +56,14 @@ def test_atomtyper_elemental(self):
         """
         Test elemental atomtype sampler for hydrogen
         """
-        atomtype_sampler = smarty.AtomTypeSamplerElemental(self.mols_alkethoh,
+        atomtype_sampler = smarty.AtomTypeSampler(self.mols_alkethoh,
                 self.basetypes, self.basetypes, self.combine_decs,
                 self.replacements, self.mols_alkethoh_ref, 0.1, False,
-                'combinatorial-decorators', '1')
+                'combinatorial-decorators', 1)
```

Comment:
@davidlmobley this is testing elemental sampling, the element is the last keyword argument so the 1 on the end sets element=1 (instead of 0). 

I'm confused about why I would have to specify element= when calling the function if I'm specifying all of the keyword arguments.